File: CloudFlow/FrontEnd-App/src/app/shared/model/reports/raspunsuri-banci-report-bundle.model.ts
import { JsonObject, JsonProperty } from "json2typescript";
import { SelectItemModel } from "@app/shared/select-item.model";

@JsonObject
export class RaspunsuriBanciReportBundleModel {

    @JsonProperty ("denumiriBanci", [SelectItemModel])
    public denumiriBanci: SelectItemModel[] = [];
    
    @JsonProperty ("proiecte", [SelectItemModel])
    public proiecte: SelectItemModel[] = [];
    
}

File: CloudFlow/src/ro/cloudSoft/cloudDoc/presentation/client/shared/model/report/RaspunsuriBanciReportBundleModel.java
package ro.cloudSoft.cloudDoc.presentation.client.shared.model.report;

import java.util.List;

import ro.cloudSoft.cloudDoc.presentation.client.shared.model.SimpleListItemModel;

public class RaspunsuriBanciReportBundleModel {

	private List<SimpleListItemModel> denumiriBanci;
	private List<SimpleListItemModel> proiecte;
	
	public List<SimpleListItemModel> getDenumiriBanci() {
		return denumiriBanci;
	}
	public void setDenumiriBanci(List<SimpleListItemModel> denumiriBanci) {
		this.denumiriBanci = denumiriBanci;
	}
	public List<SimpleListItemModel> getProiecte() {
		return proiecte;
	}
	public void setProiecte(List<SimpleListItemModel> proiecte) {
		this.proiecte = proiecte;
	}
	
}
